lib/massager_filename_generator.ex

```diff
@@ -0,0 +1,33 @@
+defmodule MassagerFilenameGenerator do
+  # For example, this will turn: "input_path/order_1.csv" into {"input_path/order_1.csv", "input_path/order_01.csv"}
+  def generate_filename_pairs(%{
```

Comment:
The use of a map as a function parameter (and matching on the contents of the map) makes it easier to add/remove/change parameters without changing the function signature (and also do function matching much easier).